### ISSUE TITLE
Arduino Encoder Program not working

### ISSUE THREAD

**Issue by willward20** [willward20/the-bot-bros#4]
I downloaded the .cpp file Dr. Zhang added and uploaded it to the Arduino. There were some things I uncommented, and I also changed some values to match our robot. The program uploads and runs fine, but the printed counter values are zero even when the motors are turning. I realized that the program never turns on the encoder Vcc pins, so I added two lines for that: `digitalWrite(A5, HIGH);` and `digitalWrite(A6, HIGH);`. This didn't fix the issue though.

**Comment by linzhangUCA** [comment]
Look like the [new code](https://github.com/willward20/the-bot-bros/blob/main/NewEncoderCounter.cpp) works fine. I'll try to debug the old one.

**Comment by willward20** [comment]
The [new code](https://github.com/willward20/the-bot-bros/blob/main/NewEncoderCounter.cpp) isn't working right now. It worked yesterday, and it worked when I tested it at the lab today at 5:30. Then, 20 minutes later it stopped working. I thought maybe I had edited the code by accident in the IDE, so I downloaded the new code again and re-uploaded it, and it still did not work. I tried rebooting the laptop and the pi. No luck. Very confused haha.
